jabkit/src/main/java/org/jabref/cli/Pseudonymize.java

```diff
@@ -0,0 +1,98 @@
+package org.jabref.cli;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.logic.pseudonymization.Pseudonymization;
+import org.jabref.logic.pseudonymization.PseudonymizationResultCsvWriter;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.database.BibDatabaseContext;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Mixin;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+import picocli.CommandLine.ParentCommand;
+
+@Command(name = "pseudonymize", description = "Anonymize the library")
+public class Pseudonymize implements Runnable {
+    private final static Logger LOGGER = LoggerFactory.getLogger(Pseudonymize.class);
+
+    @ParentCommand
+    private ArgumentProcessor argumentProcessor;
+
+    @Mixin
+    private ArgumentProcessor.SharedOptions sharedOptions = new ArgumentProcessor.SharedOptions();
+
+    @Parameters(index = "0", description = "File to be anonymized")
+    private String inputFile;
+
+    @Option(names = {"--output"}, description = "Output file")
+    private String outputFile;
+
+    @Option(names = {"--key"}, description = "Output keys file")
+    private String key;
+
+    @Override
+    public void run() {
+        String fileName = getFileName(inputFile);
+        String outputFileName = fileName + ".bib";
+        String keyFileName = fileName + "_keys.csv";
+
+        // todo: case- output file already exist !
```

Comment:
Yes, please.

DOES NOT overwrite, but errors

See https://clig.dev/#arguments-and-flags

> Confirm before doing anything dangerous. A common convention is to prompt for the user to type y or yes if running interactively, or requiring them to pass -f or --force otherwise.

Add `-f` / `--force` to force overwrite.